README.md

```diff
@@ -10,8 +10,21 @@ there. You connect HomeAssistant to it and it will do all the parsing.
 ## Wiring
 Wiring is the same as https://github.com/daniel-jong/esp8266_p1meter and also the initial inspiration of using the inverter of the hardware serial.
 
+The hardware pull-up resistor is not necessary on many boards. For example Wemos/Lolin D1 mini (lite) boards with a CH340 uart chip already have a RX pull-up activated by the bootloader.
+
+The ESP8266 can also be powered directly from the P1 port, without external power supply. The following reference wiring diagram shows how:
```

Comment:
this is only true for DSMR 5+. so this should be nuanced a bit.

Reply:
oops..